aep/general/0134/aep.md.j2

```diff
@@ -57,7 +59,28 @@ separator, `snake_case` is used.
 
 {% tab oas %}
 
-**Note:** OAS example not yet written.
+```yaml
+paths:
+  /v1/publishers/{publisher}/books:
+    patch:
+      summary: Update a book
+      operationId: publishers.book:create
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Book'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Book'
+```
+
+- The method **must** adhere to the behavior specified in [IETF 7396 - JSON Merge Patch](https://datatracker.ietf.org/doc/html/rfc7396).
```

Comment:
```suggestion
- The method **must** adhere to the behavior specified in [RFC 7396 - JSON Merge Patch](https://datatracker.ietf.org/doc/html/rfc7396).
```

Reply:
How about IETF RFC? I think that's important to make it clear what body one can find the guidance in.